PUML.h

```diff
@@ -885,6 +919,16 @@ class PUML
 		generatedSharedAndGID<face_t, edge_t, internal::Topology<Topo>::faceedges()>(m_faces, m_edges);
 	}
 
+	/**
+	 * @return The total number of cells
+	 *
+	 * @note This is the number of all cells within the mesh.
```

Comment:
or simply:
returns the total number of cells within the mesh
and remove the note.

Reply:
Haha true